engine/runtime/eval-expr/src/module/str.rs

```diff
@@ -22,6 +22,37 @@ pub(crate) mod str_module {
         let regex = Regex::new(regex).unwrap();
         regex.is_match(haystack)
     }
+
+    pub fn string_to_int(s: String) -> i64 {
+        s.parse::<i64>().unwrap()
+    }
+
+    pub fn string_to_f64(s: String) -> f64 {
```

Comment:
I think these names are inconsistent.

I feel either of the followings are consistent. What do you think?
- string_to_int, string_to_float (or string_to_double?)
- string_to_i64, string_to_f64